packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_controller.dart

```diff
@@ -539,6 +544,8 @@ class DeepLinksController extends DisposableController
           scheme: linkdata.scheme,
           associatedDomains: linkdata.associatedDomains,
           associatedPath: linkdata.associatedPath,
+          hasAndroidAssetLinksFile: ! (androidDomainErrors[linkdata.domain] ??[]) .contains(AndroidDomainError.existence),
+          hasIosAasaFile: ! (iosDomainErrors[linkdata.domain] ?? []).contains(IosDomainError.existence),
```

Comment:
Can we rewrite these as:
```
(androidDomainErrors[linkdata.domain] as List?)?.contains(AndroidDomainError.existence) ?? false
```

This will avoid dynamic execution for `.contains`